ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -34,7 +34,7 @@ cat > packet-setup.yaml <<-EOF
         project_id: "{{ packet_project_id }}"
         hostnames: "{{ packet_hostname }}"
         operating_system: centos_8
-        plan: m2.xlarge.x86
+        plan: m3.large.x86
```

Comment:
This value should be a parameter to avoid impacting other teams that are currently using `m2.xlarge.x86` and happy with that.

Reply:
(reminder note: we still have a small portion of the setup managed in the [devscripts-setup step](https://github.com/openshift/release/blob/2e6b7834df9f32341fc41a31bebd4ace7ca83e13/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L76-L82) for the nvme devices)

Reply:
> other teams that are currently using m2.xlarge.x86 and happy with that

What's the case for that? The old plan is being phased out, everyone using it will be more and more unhappy with time.

Reply:
Different teams can have different priorities in migrating their workflow/tests to different hardware (and also could have different budget allocations, as for the AI team). These steps are at least shared with the AI team (cc @lranjbar), and these changes could have some impacts on their jobs.
Adding an environment variable with a default value in the step ref is pretty straightforward, and will allow our team to change plan without impacting on the others

Reply:
Agreed with adding a parameter to pass in as environment variable. Obviously we will want to move to a new plan as well but it may not happen all at once. Also it would be nice to add this just in general because there might be workflows that require less compute power in general so maybe we might even take a look at smaller server plans in the future.

Reply:
Could someone point me where I can define environment variables per job or group of jobs? I cannot figure it out.

Reply:
They can be added in the `env` section of the step definition, for example see [baremetalds-e2e-test](https://github.com/openshift/release/blob/d85642440eb25d2c7f14b61d4042f57123423e1f/ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-ref.yaml#L37) (more details in the [openshift ci doc](https://docs.ci.openshift.org/docs/architecture/step-registry/#declaring-step-parameters))